CONTRIBUTING.md

```diff
@@ -66,6 +66,14 @@ go generate .
 ```
 This will generate subcode_generated.go from the given solClient.h
 
+And to generate ccsmp_cache_session_prop_generated.go and the other ccsmp props/enums from the given solClient.h and solCache.h, run this:
+```
+export SOLCACHE_H=/path/to/repo/lib/linux/include/solclient/solCache.h
```

Comment:
I think there isn't "/linux/" in that path.
And we'll have to change the instructions here to include solCache.h in the copy and set the new env var:
https://sol-jira.atlassian.net/wiki/spaces/CPD/pages/2868052115/Golang+Process+and+Tools#Updating-CCSMP

I updated my script just now.

Reply:
Okay. I have made the update to remove the `/linux` from the path template.